test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildANetCoreApp.cs

```diff
@@ -108,6 +109,16 @@ public void It_targets_the_right_framework_depending_on_output_type(bool selfCon
                 actualRuntimeFrameworkVersion.Should().Be(expectedRuntimeVersion);
             }
 
+            string devruntimeConfigFile = Path.Combine(outputDirectory.FullName, testProject.Name + ".runtimeconfig.dev.json");
+            if (File.Exists(devruntimeConfigFile))
```

Comment:
This should assert that the file exists, not silently ignore when the file doesn't exist.